src/nvim/ex_getln.c

```diff
@@ -985,6 +985,9 @@ static uint8_t *command_line_enter(int firstc, long count, int indent, bool init
   if (cmdheight0) {
     // Restore cmdheight
     set_option_value("ch", 0L, NULL, 0);
+
+    // Redraw is needed for command line completion
+    redraw_all_later(CLEAR);
```

Comment:
is CLEAR really needed or does something less drastic work, such as REDRAW_TOP, SOME_VALID, ..?

Reply:
Unfortunately, `CLEAR` is really needed.  Other modes does not work....
I have tested them.